### PR TITLE
Chapter 13 Microposts

### PR DESCRIPTION
**Pull request by Iskandar19**

### CODE REVIEW

**Comment by Iskandar19** [comment]
Ready

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -3,5 +3,24 @@
...
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3>Micropost Feed</h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
+...
```
Comment (huongnt-2545) [inline app/views/static_pages/home.html.erb]: you seem missing some code handler at here, please check the tutorial

**Comment by Iskandar19** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -3,8 +3,8 @@
       <%= link_to "sample app", "#", id: "logo" %>
       <nav>
         <ul class="nav navbar-nav navbar-right">
-          <li><%= link_to "Home", "#" %></li>
-          <li><%= link_to "Help", "#" %></li>
+          <li><%= link_to "Home", root_url %></li>
+          <li><%= link_to "Help", "http://localhost:3000/static_pages/help" %></li>
```
Comment (huongnt-2545) [inline app/views/layouts/_header.html.erb]:
```suggestion
          <li><%= link_to "Help", help_url %></li>
```

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -3,5 +3,23 @@
...
+      <section class="micropost_form">
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3>Micropost Feed</h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
```
Comment (huongnt-2545) [inline app/views/static_pages/home.html.erb:24]: You seem missing some code handler at here
please check

Comment (Iskandar19) [reply app/views/static_pages/home.html.erb:24]: I think there is no more code there, I have rechecked with the reference slides, videos and compare with another course mate too.

**Comment by Iskandar19** [comment]
ready

File: app/views/static_pages/home.html.erb (hunk 3 above)
Comment (Iskandar19) [reply app/views/static_pages/home.html.erb:24]: Screenshot from 2021-09-09 15-19-35

Comment (Iskandar19) [reply app/views/static_pages/home.html.erb:24]: I have double checked that no code were added after this point either in this chapter, though some addition will be made later on chapter 14 and my sample app has been working perfectly fine.

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**